openrtb/openrtbtest/testutil.go

```diff
@@ -103,6 +103,10 @@ func getNumOfJsonFields(modelType reflect.Type) (result int) {
 	n := modelType.NumField()
 
 	for i := 0; i < n; i++ {
+		if modelType.Field(i).Type.Kind() == reflect.Struct {
```

Comment:
This would not work if you have something like
```go
type A struct {
   SomeStruct // this is okay
  SomeOtherStruct `json:"field"` // this would not work.
}
```